test/node-report/test-exception.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+// Testcase to produce report on uncaught exception
+require('../common');
```

Comment:
Can you make sure the tests pass when `./configure --without-node-report` was used? I’m sorry if this directory is skipped entirely and I don’t see it, but for most compile-time optional features we have some specific `common.skipIfFooDisabled()` method or so…

Reply:
sure, will look at this - yes, I did not use any special settings to skip tests with feature off, so I am sure the tests will run but fail at present. will fix it.

Reply:
Done. Now the tests are skipped if node report is disabled at the source level. Added a helper method skipIfReportDisabled() in common/index.js to support this functionality.